app/components/stepper-signup.js

```diff
@@ -14,9 +14,15 @@ export default class StepperSignupComponent extends Component {
   @tracked signupDetails = {
     firstname: '',
     lastname: '',
+    username: '',
   };
 
   @action handleInputChange(key, value) {
     set(this.signupDetails, key, value);
   }
+
+  @action setUsername(generateUsername) {
+    this.signupDetails.username = generateUsername;
+    console.log(this.signupDetails);
```

Comment:
remove this console.log

Reply:
Done with changes can you please check my latest commit